mkdocs-cn/mkdocs.yml

```diff
@@ -53,203 +53,16 @@
-        'builders/interact/eth-libraries/send-transaction.md': 'builders/build/eth-api/libraries/index.md'
-        'builders/tools/eth-libraries/web3js.md': 'builders/build/eth-api/libraries/web3js.md'
-        'builders/tools/eth-libraries/web3py.md': 'builders/build/eth-api/libraries/web3py.md'
-        'builders/tools/pubsub.md': 'builders/build/eth-api/pubsub.md'
-        'builders/tools/verify-contracts/block-explorers.md': 'builders/build/eth-api/verify-contracts/block-explorers.md'
-        'builders/tools/verify-contracts/etherscan-plugins.md': 'builders/build/eth-api/verify-contracts/etherscan-plugins.md'
-        'builders/tools/sidecar.md': 'builders/build/substrate-api/sidecar.md'
-        'builders/tools/explorers.md': 'builders/get-started/explorers.md'
-        'builders/get-started/moonbase.md': 'builders/get-started/networks/moonbase.md'
         'builders/get-started/moonriver.md': 'builders/get-started/networks/moonriver.md'
-        'builders/get-started/moonbeam.md': 'builders/get-started/networks/moonbeam.md'
-        'builders/get-started/moonbeam-dev.md': 'builders/get-started/networks/moonbeam-dev.md'
-        'builders/tools/wallets/metamask.md': 'builders/integrations/wallets/metamask.md'
-        'builders/tools/wallets/walletconnect.md': 'builders/integrations/wallets/walletconnect.md'
-        'builders/interact/identity.md': 'tokens/manage/identity.md'
-        'builders/tools/multisig-safe.md': 'tokens/manage/multisig-safe.md'
-        'builders/interact/proxy-accounts.md': 'tokens/manage/proxy-accounts.md'
-        'builders/tools/eth-libraries/etherjs.md': 'builders/build/eth-api/libraries/ethersjs.md'
-        'builders/interact/scaffold-eth.md': 'builders/build/eth-api/dev-env/scaffold-eth.md'
-        'builders/build/canonical-contracts/precompiles/index.md': 'builders/pallets-precompiles/precompiles/index.md'
-        'builders/build/canonical-contracts/precompiles/staking.md': 'builders/pallets-precompiles/precompiles/staking.md'
-        'builders/build/canonical-contracts/precompiles/democracy.md': 'builders/pallets-precompiles/precompiles/index.md'
-        'builders/build/canonical-contracts/precompiles/erc20.md': 'builders/pallets-precompiles/precompiles/erc20.md'
-        'builders/build/canonical-contracts/precompiles/eth-mainnet.md': 'builders/pallets-precompiles/precompiles/eth-mainnet.md'
-        'builders/build/canonical-contracts/contracts.md': 'builders/build/canonical-contracts.md'
-        'builders/pallets-precompiles/precompiles/democracy.md': 'builders/pallets-precompiles/precompiles/index.md'
-        'builders/integrations/bridges/axelar.md': 'builders/interoperability/protocols/axelar.md'
-        'builders/integrations/bridges/index.md': 'builders/interoperability/protocols/index.md'
-        'builders/xcm/fees.md': 'builders/interoperability/xcm/fees.md'
+        'builders/get-started/networks/layer2/bobabeam.md': 'builders/get-started/networks/index.md'
```

Comment:
```suggestion
        'builders/get-started/networks/layer2/bobabeam.md': 'builders/get-started/networks/'
```

Reply:
Not sure if this is the right formatting but https://docs.moonbeam.network/builders/get-started/networks/index/ 404s